source/IntegrationTests/Behaviours/IncomingRequests/GivenAggregatedMeasureDataV2RequestWithDelegationTests.cs

```diff
@@ -495,114 +553,129 @@ await ThenNotifyAggregatedMeasureDataDocumentIsCorrect(
             new(originalActor.ActorNumber, originalActor.ActorRole),
             new(delegatedToActor.ActorNumber, delegatedToActor.ActorRole),
-            "609",
+            "543",
             ProcessType.RequestEnergyResults,
             GetNow());
 
+        // Act
         await GivenReceivedAggregatedMeasureDataRequest(
             documentFormat: incomingDocumentFormat,
             senderActorNumber: delegatedToActor.ActorNumber,
             senderActorRole: originalActor.ActorRole,
-            meteringPointType: MeteringPointType.Consumption,
-            settlementMethod: SettlementMethod.Flex,
-            periodStart: (2024, 1, 1),
-            periodEnd: (2024, 1, 31),
+            meteringPointType: testMessageData.ExampleMessageData.MeteringPointType,
+            settlementMethod: testMessageData.ExampleMessageData.SettlementMethod,
+            periodStart: (2022, 1, 1),
+            periodEnd: (2022, 2, 1),
             energySupplier: energySupplierNumber,
             balanceResponsibleParty: balanceResponsibleParty,
-            new (string? GridArea, TransactionId TransactionId)[]
-            {
-                (null, TransactionId.From("12356478912356478912356478912356478")),
-            });
-
-        // Act
-        await WhenAggregatedMeasureDataProcessIsInitialized(senderSpy.LatestMessage!);
+            new (string? GridArea, TransactionId TransactionId)[] { (null, transactionId), });
 
         // Assert
-        var message = await ThenAggregatedTimeSeriesRequestServiceBusMessageIsCorrect(
+        var message = ThenRequestCalculatedEnergyTimeSeriesInputV1ServiceBusMessageIsCorrect(
             senderSpy,
-            new AggregatedTimeSeriesMessageAssertionInput(
-                GridAreas: new List<string>() { "512", "609" },
-                RequestedForActorNumber: originalActor.ActorNumber.Value,
-                RequestedForActorRole: originalActor.ActorRole.Name,
-                EnergySupplier: energySupplierNumber.Value,
-                BalanceResponsibleParty: balanceResponsibleParty.Value,
-                BusinessReason: BusinessReason.BalanceFixing,
-                Period: new Period(CreateDateInstant(2024, 1, 1), CreateDateInstant(2024, 1, 31)),
-                SettlementVersion: null,
-                SettlementMethod: SettlementMethod.Flex,
-                MeteringPointType: MeteringPointType.Consumption));
-
-        // TODO: Assert correct process is created
+            new RequestCalculatedEnergyTimeSeriesInputV1AssertionInput(
+                transactionId,
+                originalActor.ActorNumber.Value,
+                originalActor.ActorRole.Name,
+                BusinessReason.BalanceFixing,
+                PeriodStart: CreateDateInstant(2022, 1, 1),
+                PeriodEnd: CreateDateInstant(2022, 2, 1),
+                energySupplierNumber?.Value,
+                balanceResponsibleParty?.Value,
+                new List<string>() { "542", "543" },
```

Comment:
```suggestion
                gridAreasWithDelegations,
```
if you'r doing the above